Clarify : Backup of Azure VMs / AWS EC2 Is Not Supported by MABS (#127617)
Azure/azure-content: articles/backup/backup-support-matrix-mabs-dpm.md

--- | --- | ---
**Back up on-premises machines/workloads** | DPM/MABS protection agent runs on the machines that you want to back up.<br/><br/> The MARS agent on DPM/MABS server.<br/> The minimum version of the Microsoft Azure Recovery Services agent, or Azure Backup agent, required to enable this feature is 2.0.8719.0.  | DPM/MABS must be running on-premises.

> [!NOTE]

> > ## Backup of Azure VMs / AWS EC2 and others... Is Not Supported by MABS

Reviewer: ```suggestion
> Backup of virtual machines hosted on public cloud platforms such as Azure VMs or AWS EC2 using DPM/MABS is not supported.
```

> > A frequently asked question is whether virtual machines hosted on public cloud platforms such as Azure VMs or AWS EC2 can be backed up using DPM/MABS. The answer is that **these environments are not supported**.

Reviewer 2: As it's a note in official documentation, preferably use another tone
```suggestion
> Backup of Virtual Machines hosted on public cloud platforms (e.g., Azure, AWS, GCP) is **not supported by MABS**.
```

Author: @AkshitGureja 
Good suggestion, this version is much clearer and concise. I agree 👍

> > A frequently asked question is whether virtual machines hosted on public cloud platforms such as Azure VMs or AWS EC2 can be backed up using DPM/MABS. The answer is that **these environments are not supported**.

Reviewer: ```suggestion
> 
```

> >

Reviewer: ```suggestion
> Also, Bare Metal Recovery (BMR) with MABS is supported only for recovery on the same hardware; recovery to different hardware or cloud environments (such as Azure VM or AWS EC2) isn't supported.
```

> > Bare Metal Recovery (BMR) with MABS is **only supported for recovery on the same hardware**. Recovery to different hardware or in cloud environments (e.g., Azure VM / AWS EC2) is **not supported**.

Reviewer 2: ```suggestion
> Bare Metal Recovery (BMR) with MABS is **only supported for recovery on the same hardware**. Recovery to different hardware or cloud environments (e.g., Azure, AWS, GCP) is **not supported**.
```

Author: @AkshitGureja 
Good suggestion, this version is much clearer and concise. I agree 👍

> > Bare Metal Recovery (BMR) with MABS is **only supported for recovery on the same hardware**. Recovery to different hardware or in cloud environments (e.g., Azure VM / AWS EC2) is **not supported**.

Reviewer: ```suggestion
>

>

> > ### Support Matrix

Reviewer 2: @v-dirichards Do we support tables within notes. If not, we can remove this as the line above conveys the same information.

Author: If tables are supported within notes, I’d suggest keeping it.  
It makes the content clearer and easier for readers to compare at a glance. 👍

Reviewer 3: @akshitgureja @aktsmm Yes, tables in notes are allowed.

Would you add a comment to indicate what action to take on this PR? Thanks!

Author: @v-dirichards 

Thanks for the reply.
I'll leave the decision to @AkshitGureja

Reviewer 2: @v-dirichards Let's proceed ahead with keeping the table then :)

>
> | Scenario | Supported |
> | --- | --- |
> | Recover system state after BMR on the same hardware | ✅ Yes |
> | Recover system state after BMR on different hardware | ❌ No |
> | Recover system state after non-BMR full restore (same/different hardware) | ❌ No |
>

> > This limitation is primarily due to the fact that **system state backups contain hardware-dependent information**, making recovery on different environments technically infeasible.

Reviewer: ```suggestion
> This limitation is primarily due to the fact that **system state backups contain hardware-dependent information**, making recovery on different environments technically infeasible.
```

Reviewer 3: @aktsmm  - Can you respond to the requested changes or let us know if you would like us to commit them?

@Akshitgureja @AbhishekMallick-MS  - When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

Author: @v-dirichards 
@AbhishekMallick-MS 
Sounds good to me, thanks for the suggestion! 👍
